staging/src/k8s.io/client-go/util/workqueue/queue.go

```diff
@@ -55,7 +55,13 @@ func newQueue(c clock.Clock, metrics queueMetrics, updatePeriod time.Duration) *
 		metrics:                    metrics,
 		unfinishedWorkUpdatePeriod: updatePeriod,
 	}
-	go t.updateUnfinishedWorkLoop()
+
+	// Don't start the goroutine for a type of noMetrics so we don't consume 
```

Comment:
I think the gofmt test is complaining about a trailing space in this line: 
```
-	// Don't start the goroutine for a type of noMetrics so we don't consume 
+	// Don't start the goroutine for a type of noMetrics so we don't consume
```

Reply:
Removed it and repushed.